elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../pfe-sass/pfe-sass";
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-indeterminate]) {
+    animation: spin 1s linear infinite;
+    -webkit-animation:spin 1s linear infinite;
+    -moz-animation:spin 1s linear infinite;
+
+    border-bottom: 4px solid rgba(0,0,0,0.25);
```

Comment:
Can we set local variables for these so they aren't repeated? Maybe something like:
```
$pfe-progress-indicator--border-light: 4px solid rgba(0,0,0,0.25);
$pfe-progress-indicator--border-dark: 4px solid rgba(0,0,0,0.75);
```